content/docs/plugins/notifications-plugin/_index.md

```diff
@@ -2,48 +2,42 @@
 title: "Notifications Plugin"
 ---
 
-Documentation section of notifications plugin.
+Documentation section of notifications plugin. This plugin is part of Backstage Core.
 
 # Frontend
-The frontend provides both UI and API.  
 The frontend UI is implemented as a side bar item.  
-The following screenshots shows how to use the notifications UI.  
-Whenever a new system notification (update) is detected the frontend pops an alert to the user.  
-
-## Installation and API
-Refer to the plugin's [README](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/notifications/README.md)
+The following screenshots show how to use the notifications UI.  
+Whenever a new notification is detected the frontend pops an alert to the user.  
 
 ## The Frontend UI
 
 ### Notifications page
-![Frontend UI](./fe-screenshot.png)
-
-### New messages icon
-![New messages icon](./new-messages-icon.png)
+![Frontend UI](./sidebar-button-plus-alert.png)
 
-### Filter and sort
-![Filter and sort](./sort-filter.png)
+### Notifications list
+![Notifications list](./list-view.png)
 
-### Updates
-![Updates](./updates.png)
+# Backend
+The backend plugin provides the backend application for reading and writing notifications.
 
-### Update alert
-When a new update notification is received the UI pops an alert
-  
-![Update alert/popup](./system%20alert.png)
+## REST endpoints and OpenAPI
+The plugin does not yet provide an OpenAPI spec. However, we created our own spec for posting notifications. It can be found [here](https://github.com/parodos-dev/serverless-workflows/blob/main/shared/specs/notifications-openapi.yaml).
+The readme file provides an [example](https://github.com/backstage/backstage/tree/master/plugins/notifications-backend#sending-notifications-by-external-services) for posting a notification.
 
-### Notifications list
-![Notifications list](./notifications-list.png)
+# Forward to Email
+It is possible to forward notification content to email address. In order to do that you must add the [Email Processor Module](https://github.com/backstage/backstage/tree/master/plugins/notifications-backend-module-email) to your backstage backend. The email title is taken from the notification title and the email body is taken from the notification's description and link.
 
-# Backend
-The backend plugin provides the backend application for reading and writing notifications.  
-It also provides and OpenAPI spec for generating clients in various languages.
+## Configuration
+An example configuration, as well as a link to all configuration options, can be found in the module's [readme](https://github.com/backstage/backstage/tree/master/plugins/notifications-backend-module-email).
 
-## Installation
-Refer to the plugin's [README](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/notifications-backend/README.md)
+## Ignoring unwanted notifications
```

Comment:
should be this a subsection of `Configuration`?

Reply:
it is an important feature that we want to emphasize. it deserves a section